README.md

```diff
@@ -1 +1,167 @@
+- **Delete Book button**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Our live demo can be found on
+
+- [Live Demo Link](https://leehaney254.github.io/AwsomeBooks/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Create a local directory where you can clone the project.
+- Clone the project into your repository
+- Install the dependencies
+- Open the project on a browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and GitHub
+- A code editor
+- Nodejs
+- A browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+- Create a local directory where you can clone the project.
+- Clone the project into your repository
+- Install the dependencies
+- Open the project on a browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Just run npm i to install all dependencies
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+**Leehaney George**
+
+- GitHub: [@githubhandle](https://github.com/leehaney254)
+- Twitter: [@twitterhandle](https://twitter.com/Lee06785586)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/leehaney-george-0a4a51178/)
+
+**Steven Mukami**
+
+- GitHub: [@githubhandle](https://github.com/stevenmukama)
+- Twitter: [@twitterhandle](https://twitter.com/mukama_steven)
+
+**Muhammad Talha Hanif**
+
+- GitHub: [@githubhandle](https://github.com/Talha-Hanif5666)
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Implement nav-bar**
+- [ ] **Implement more pages**
+- [ ] **Add functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/leehaney254/Castapone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly leave a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for the great resources shared.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] You have a link of your license but this link is not working. 😨 kindly add a license file using these [guidelines ](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)